modules/ansible/ansibleee.go

```diff
@@ -0,0 +1,321 @@
+		job.Spec.Template.Spec.InitContainers = a.InitContainers
+	}
+	if len(a.ServiceAccountName) > 0 {
+		job.Spec.Template.Spec.ServiceAccountName = a.ServiceAccountName
+	}
+	// Set primary inventory if specified as string
+	existingInventoryMounts := ""
+	if len(a.Inventory) > 0 {
+		setRunnerEnvVar(h, "RUNNER_INVENTORY", a.Inventory, "inventory", job, hashes)
+		existingInventoryMounts = "/runner/inventory/inventory.yaml"
+	}
+	// Report additional inventory paths mounted as volumes
+	// AnsibleEE will later attempt to use them all together with the primary
+	// If any of the additional inventories uses location of the primary inventory
+	// provided by the dataplane operator raise an error.
+	if len(a.ExtraMounts) > 0 {
+		for _, inventory := range a.ExtraMounts {
+			for _, mount := range inventory.Mounts {
+				// Report when we mount other inventories as that alters ansible execution
+				if strings.HasPrefix(mount.MountPath, "/runner/inventory/") {
+					h.GetLogger().Info(fmt.Sprintf("additional inventory %s mounted", mount.Name))
+					if searchIndex := strings.Index(existingInventoryMounts, mount.MountPath); searchIndex != -1 {
+						return nil, fmt.Errorf(
+							"inventory mount %s overrides existing inventory location",
+							mount.Name)
+					}
+					existingInventoryMounts = existingInventoryMounts + fmt.Sprintf(",%s", mount.MountPath)
+				}
+			}
+		}
+	}
+
+	if len(a.PlaybookContents) > 0 {
+		setRunnerEnvVar(h, "RUNNER_PLAYBOOK", a.PlaybookContents, "playbookContents", job, hashes)
+	} else if len(playbook) > 0 {
+		// As we set "playbook.yaml" as default
+		// we need to ensure that PlaybookContents is empty before adding playbook
+		setRunnerEnvVar(h, "RUNNER_PLAYBOOK", playbook, "playbooks", job, hashes)
+	}
+
+	if len(a.CmdLine) > 0 {
+		setRunnerEnvVar(h, "RUNNER_CMDLINE", a.CmdLine, "cmdline", job, hashes)
+	}
+	if len(a.Labels["deployIdentifier"]) > 0 {
+		hashes["deployIdentifier"] = a.Labels["deployIdentifier"]
+	}
+
+	a.addMounts(job)
+
+	// if we have any extra vars for ansible to use set them in the RUNNER_EXTRA_VARS
+	if len(a.ExtraVars) > 0 {
+		keys := make([]string, 0, len(a.ExtraVars))
+		for k := range a.ExtraVars {
+			keys = append(keys, k)
+		}
+		sort.Strings(keys)
+		parsedExtraVars := ""
+		// unmarshal nested data structures
+		for _, variable := range keys {
+			var tmp interface{}
+			err := yaml.Unmarshal(a.ExtraVars[variable], &tmp)
+			if err != nil {
+				return nil, err
+			}
+			parsedExtraVars += fmt.Sprintf("%s: %s\n", variable, tmp)
+		}
+		setRunnerEnvVar(h, "RUNNER_EXTRA_VARS", parsedExtraVars, "extraVars", job, hashes)
+	}
+
+	hashPodSpec(h, podSpec, hashes)
+
+	_, errorHash := hashOfInputHashes(hashes)
+	if errorHash != nil {
+		return nil, fmt.Errorf("error generating hash of input hashes: %w", errorHash)
+	}
+
+	return job, nil
+}
+
+// labelsForOpenStackAnsibleEE returns the labels for selecting the resources
+// belonging to the given openstackansibleee CR name.
```

Comment:
I think we don't have an `openstackansibleee` CR anymore, right? I assume this comment was copy/paste from the AEE operator

Reply:
Ah, yes, true. This will need to be changed